cmd/juju/application/deployer/bundle.go

```diff
@@ -215,7 +215,11 @@ type charmstoreBundle struct {
 
 // String returns a string description of the deployer.
 func (d *charmstoreBundle) String() string {
-	return fmt.Sprintf("deploy charm store bundle: %s", d.bundleURL.String())
+	str := fmt.Sprintf("deploy charm store bundle: %s", d.bundleURL.String())
+	if d.origin == (commoncharm.Origin{}) {
+		return str
+	}
+	return fmt.Sprintf("%s with origin: %s", str, d.origin.CoreChannel().String())
```

Comment:
Can you double-check with the other places where we log the origin and ensure we use consistent terminology across the board? IIRC we tend to use "channel: X" in this package.

Reply:
Sure can!

Reply:
Have a look at bundlehandler.go:338 and refresh.go:370